lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
I think that might be confusing to some users; what about something like:
> Allow theme to use default wallpaper

Sorry to add more fuel to this fire...  :-P